src/Service/Windows/OrbitGrpcServer.cpp

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitGrpcServer.h"
+
+#include <absl/flags/declare.h>
+#include <absl/flags/flag.h>
+#include <grpcpp/security/server_credentials.h>
+#include <grpcpp/server_impl.h>
+
+#include <string>
+#include <utility>
+
+#include "CaptureService/CaptureStartStopListener.h"
+#include "WindowsCaptureService/WindowsCaptureService.h"
+#include "WindowsProcessService/ProcessServiceImpl.h"
+#include "grpcpp/ext/proto_server_reflection_plugin.h"
+#include "grpcpp/grpcpp.h"
+#include "grpcpp/health_check_service_interface.h"
+
+ABSL_DECLARE_FLAG(bool, devmode);
+
+namespace orbit_service {
+
+using orbit_capture_service::CaptureStartStopListener;
+using orbit_windows_capture_service::WindowsCaptureService;
+using orbit_windows_process_service::ProcessServiceImpl;
+
+namespace {
+
+class OrbitGrpcServerImpl final : public OrbitGrpcServer {
```

Comment:
This file is basically the same as the Linux version. It only differs the the members of the class and in the calls to `builder.RegisterService`. Fixes and changes on one side will end up not going to the other side. We need to share code here.